indispensable/src/jvmTest/kotlin/at/asitplus/signum/indispensable/TagEncodingTest.kt

```diff
@@ -0,0 +1,89 @@
+@file:OptIn(ExperimentalStdlibApi::class)
+
+package at.asitplus.signum.indispensable
+
+import at.asitplus.signum.indispensable.asn1.*
+import io.kotest.assertions.withClue
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+import io.kotest.property.Arb
+import io.kotest.property.arbitrary.int
+import io.kotest.property.arbitrary.uInt
+import io.kotest.property.arbitrary.uLong
+import io.kotest.property.checkAll
+import org.bouncycastle.asn1.ASN1Integer
+import org.bouncycastle.asn1.DERTaggedObject
+
+class TagEncodingTest : FreeSpec({
+
+    "Manual" -{
+        withData(207692171uL, 128uL, 36uL, 16088548868045964978uL, 15871772363588580035uL) {
```

Comment:
Why not use some random generator function?

Reply:
It says "Manual" for a reason ;-) now that everything is working, the generated ones below suffice